### ISSUE TITLE
Built-in highlight does not show up on Shield connection

### ISSUE THREAD

**Issue by ijdoc** [KomodoOpenLab/TeclaAccess#164]
Consider emulating a couple of key events immediately after Shield connection to take the phone out of touch mode.

**Comment by sagar-chandarana** [comment]
I guess by highlighting we mean the scanning of navigation keyboard should be started automatically.

**Comment by ijdoc** [comment]
@sagarchandarana123 nope, this is the built-in Android highlight. This issue has not been confirmed.
